<launch>
  <node pkg="modulair_apps_python" name="object_testing" type="object_testing.py" />
</launch>